README.md

```diff
@@ -0,0 +1,321 @@
+2. reduce boilerplate around coordination, making it faster to compose existing tools to build new products
+3. increase visibility into the steps in a pipeline
+4. encourage simplicity in system design
+
+How is Captain different from other workflow engines?
+1. Captain relies on you to handle your persistence of a request and its metadata. You don't have to set up special DB for captain. For some this will be really valuable and others it will be a deal breaker.
+2. Captain is really easy to set up. We've provided an [ExampleCaptainWorkflow](https://github.com/LiveRamp/captain/tree/master/src/main/java/com/liveramp/captain/exception/ExampleCaptainWorkflow.java)
+3. Like [daemon_lib](https://github.com/LiveRamp/daemon_lib), the core library has no infrastructure dependencies beyond access to a working directory on disk.
+4. It's opinionated towards linear workflows. 
+
+### Adding the dependency
+
+In Maven, make your project section look like this:
+
+```
+<project>
+<!-- All the other stuff -->
+
+  <dependencies>
+    <!-- All your other dependencies -->
+    <dependency>
+      <groupId>com.liveramp</groupId>
+      <artifactId>captain</artifactId>
+      <version>1.0-SNAPSHOT</version>
+    </dependency>
+  </dependencies>
+
+  <repositories>
+    <repository>
+      <id>liveramp-repositories</id>
+      <name>Liveramp Repositories</name>
+      <url>http://repository.liveramp.com/artifactory/liveramp-repositories</url>
+      <snapshots>
+        <enabled>true</enabled>
+        <updatePolicy>always</updatePolicy>
+      </snapshots>
+    </repository>
+  </repositories>
+</project>
+```
+
+The repository section is necessary because this project has not been published to Maven Central yet.
+
+### Getting Started
+
+Captain requires only 3 inputs from the developer (4 if you are running it in a distributed fashion). We're going to focus on the barest bones approach in this section of to see how we can run Captain on a single node. In later sections we cover all of the other features of Captain and discuss more complicated use cases.
+
+1. manifest: a list of the steps a request should follow in your workflow 
+2. config producer: a way for captain to find requests to process 
+3. request updater: a way for captain to interact with your persistence layer
+
+All it takes to get Captain running is the following:
+e.g.
+```java
+CaptainBuilder.of("example-captain-daemon", configProducer, manifestManager, requestUpdater)
+        .setNotifier(daemonNotifier)
+        .setConfigWaitTime(1, TimeUnit.SECONDS)
+        .setNextConfigWaitTime(1, TimeUnit.SECONDS)
+        .build()
+        .run();
+```
+You can find the rest of this example in ExampleCaptainWorkflow](https://github.com/LiveRamp/captain/tree/master/src/main/java/com/liveramp/captain/exception/ExampleCaptainWorkflow.java).
+
+Let's walk through what each of the components we're passing into this builder are.
+
+#### Config Producer
```

Comment:
idk if in this section, but i think we should explain what captain does when it picks up a request in each captain status